### PR TITLE
Create snapshot test for testing UI rendering of library components

### PR DESCRIPTION
**Pull request by monikaprabhu**
Create snapshot tests for checking UI rendering of library components. It includes 

- UIOutputComparisonTests.tsx : Example to write snapshot test by using   method ".toMatchSnapshot()" 
- Output file UIOutputComparisonTests.tsx.snap file for future comparison of test results.
- Configuration to package.json for serializing the output for creation of snapshot
- Documentation for snapshot testing added in the wiki  - https://github.com/DynamoDS/librarie.js/wiki/Author-and-run-tests-for-librarie.js
 
Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions), and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### CODE REVIEW

**Comment by monikaprabhu** [comment]
@fanwgwg, @Benglin -  please review the PR.

File: __tests__/Snapshottest/UIOutputComparisonTests.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+//import * as TestUtils from 'react-dom/lib/ReactTestUtils';
```
Comment (fanwgwg) [inline __tests__/Snapshottest/UIOutputComparisonTests.tsx]: Can we remove this line?

File: __tests__/Snapshottest/UIOutputComparisonTests.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import * as LibraryEntryPoint from '../../src/entry-point';
+import { LibraryContainer } from '../../src/components/LibraryContainer';
+import { LibraryItem } from '../../src/components/LibraryItem';
+import { ItemData } from "../../src/LibraryUtilities";
+import * as renderer from 'react-test-renderer';
```
Comment (fanwgwg) [inline __tests__/Snapshottest/UIOutputComparisonTests.tsx]: This rendered doesn't seem to be used anywhere...

**Comment by monikaprabhu** [comment]
@fanwgwg , I've made the changes required in PR comments.

File: __tests__/Snapshottest/UIOutputComparisonTests.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import * as LibraryEntryPoint from '../../src/entry-point';
+import { LibraryContainer } from '../../src/components/LibraryContainer';
+import { LibraryItem } from '../../src/components/LibraryItem';
+import { ItemData } from "../../src/LibraryUtilities";
+import { shallow } from 'enzyme';
+import { mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
```
Comment (fanwgwg) [inline __tests__/Snapshottest/UIOutputComparisonTests.tsx]: This new module should appear in `package.json`

Comment (monikaprabhu) [inline __tests__/Snapshottest/UIOutputComparisonTests.tsx]: @fanwgwg  Its already there in the above commit - https://github.com/DynamoDS/librarie.js/pull/89/commits/26a1bb5f8c8354ebc0532fae073fd2be115d4541

image

**Comment by fanwgwg** [comment]
@monikaprabhu This looks good to me!

**Pull request merged**